src/style.css

```diff
@@ -1 +1,56 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
```

Comment:
It's recommended to be consistent with margins to avoid potential margin collapse issues. Here you're using `margin-right` to space out `.stars__star`, which is fine, but ensure that you're consistently using either margin-right or margin-left throughout your project to space similar elements.